fileio/private/xsens_mvnx.m

```diff
@@ -71,6 +71,10 @@
   hdr = [];
   hdr.label = {}; 
   hdr.chanunit={};
+  hdr.chantype={};
+  hdr.chansource={};
```

Comment:
the header that is returned by `ft_read_header` adheres to a strict [API](https://en.wikipedia.org/wiki/API) to ensure that all FieldTrip functions, but also external toolboxes that use the fileio functions (like SPM amd EEGLAB) know which fields they can expect. Only fields that are common to all file formats should be in the header itself. Low-level details that apply only to a single file format can be in `hdr.orig`.

Reply:
chansource, chancomponent and chanrefspace cannot be represented like this.

Reply:
Hmmm, but when I put the chansource, chancomponent and chanrefspace in hdr.orig, it is kicked out again by `ft_preprocessing` in line 631:
```
    % don't keep hdr.orig in the output data if it is too large
    % hdr.orig can be large when caching data from specific file formats, such as bci2000_dat and mega_neurone
    if isfield(hdr, 'orig')
      s = hdr.orig;
      s = whos('s');
      if s.bytes>10240
        hdr = rmfield(hdr, 'orig');
      end
    end
```

Reply:
Have a look at https://www.fieldtriptoolbox.org/faq/how_are_the_various_data_structures_defined/ which explains two FT principles:

1. we have a small number of structures (raw, timelock, freq, etc) 
2. we try to keep the structures themselves small

With the 2nd point we more resemble SPM (which always writes data at any level of processing to a nifti file on disk) than EEGLAB (which maintains an elaborate data structure) or BrainStorm (which maintains a whole database on disk).  

If you consider my step 2 and 4 from the proposed example, you'll see that those are also not solved by a mvnx specific fieds in the header. Hence, the `hdr.orig` strategy does not result in a generic solution.

Reply:
So would you think then that it might be a better solution if the researcher would always specify the motion channels with `cfg.channels` for `data2bids`?